.github/workflows/automated_release.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
I don't have a lot of experience with tags; do we need to specify that this is only for the `main` branch or will we be tagging main directly?

Reply:
Tags are tied to commits rather than branches.